maintainers/team-list.nix

```diff
@@ -950,6 +950,15 @@ with lib.maintainers;
     shortName = "Serokell employees";
   };
 
+  stdenv = {
+    # TODO: https://github.com/NixOS/nixpkgs/issues/347955
+    members = [ ];
+    scope = "Maintain the standard environment and its surrounding logic.";
+    shortName = "stdenv";
+    enableFeatureFreezePing = true;
```

Comment:
I suspect that this will never make much sense. The stdenv changes end up being performed in staging, and roughly ~never make the release notes for a release.

Reply:
I added it just in case 😸, we can always remove it if enough people object.

Reply:
There can conceivably be release blockers relating to problems with the standard environment, so it seems fine to me.